lib/gzip_compress.c

```diff
@@ -66,7 +70,14 @@ libdeflate_gzip_compress(struct libdeflate_compressor *c,
 		xfl |= GZIP_XFL_SLOWEST_COMRESSION;
 	*out_next++ = xfl;
 	/* OS */
-	*out_next++ = GZIP_OS_UNKNOWN;	/* OS  */
+	*out_next++ = GZIP_OS_UNKNOWN;	
+	/* FILENAME */
```

Comment:
`/* Original file name (zero terminated) */`, to match the text in the gzip specification and the comment in gzip_decompress.c.